Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/attendee/checkins/items/CheckInsItem.kt

```diff
@@ -1,5 +1,8 @@
 package de.rki.coronawarnapp.ui.eventregistration.attendee.checkins.items
 
+import de.rki.coronawarnapp.eventregistration.checkins.CheckIn
 import de.rki.coronawarnapp.util.lists.HasStableId
 
-interface CheckInsItem : HasStableId
+interface CheckInsItem : HasStableId {
+    val checkin: CheckIn
```

Comment:
The CameraPermission CardItem is also a `CheckIns(List)Item` and does not have a `checkIn: CheckIn`.

Reply:
Yes, I face this now :)